scripts/update_assets/get_assets.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import csv
+import requests
+from urllib.parse import urljoin
+
+DEFAULT_API_URL = 'https://se-api.holmsecurity.com/v1/'
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='API key of the sc account')
+
+    parser.add_argument('-k',
+                        '--key-token',
+                        type=str,
+                        help='Token needed to access sc-dev',
+                        required=True)
+    parser.add_argument(
+        '-u',
+        '--url',
+        help=f"API URL to use, default is set to {DEFAULT_API_URL}",
+        default=DEFAULT_API_URL)
+    return parser.parse_args()
+
+
+def get_asset_request(args):
+    """
+    gets data from the endpoint and rewrites it to desired format.
+    """
+
+    url = urljoin(args.url, 'net-scans/assets')
+    headers = {"Authorization": f"TOKEN {args.key_token}"}
+    response = requests.get(url=url, headers=headers)
+    response.raise_for_status()
+    list_values = [v for v in response.json().values()]
```

Comment:
If you wanted to get the list of asset results, the easier way to do it would be just to:

```
resp_dict = response.json()
assets = resp_dict['results']
```

Reply:
Also there is one thing missing here - there can be more than one page of assets if we have bigger customer. You can check this place for reference: https://github.com/holmsecurity/api-examples/blob/61568045ed7b3e8672a01c66ee5000155e14b3f5/scripts/get_scan/utils.py#L14

Olaf added offset parameter to go through all pages of assets until list is exhausted. Please try similar solution!

Reply:
Done!